src/Http/Exception/RedirectException.php

```diff
@@ -74,21 +69,12 @@ public function addHeaders(array $headers)
      *
      * @param string $key The header to remove.
      * @return $this
+     * @deprecated 4.2.0 Use `setHeaders()` instead.
```

Comment:
These methods should emit runtime warnings so we don't have to go back and add them later.